api/v1/words/get.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
```

Comment:
We should be very careful in writing this route. Because, this is going to be the most used route in our app. May be we will enhance things here, while doing the UI.